content/posts/2021/pangeo-goes-live/index.md

```diff
@@ -0,0 +1,49 @@
+
+The first Pangeo cloud JupyterHub (pangeo.pydata.org; now defuct) was deployed for the [2017 American Meteoroligical Society Meeting](https://annual.ametsoc.org/2017/); since then, the Pangeo community has iterated through several different versions of prototype cloud-based hubs.
+This allowed for many new workflows that enabled a more open and collaborative pathway to doing world class research, and included access to datasets and computational resources that were previously unattainable.
+Pangeo achieved this by working in partnership with open source communities and building technology that leveraged modular open source components for their platform.
+
+In the last several years, Pangeo have built a thriving community of practice around this infrastructure.
+However as the community has grown, so has the need for more reliable and dedicated operational and developmental support since parts of the Pangeo stack require dedicated expertise and attention to managed.
+Modern scalable cloud infrastructure is one example of this. Maintaining a complex JupyterHub with many users is a difficult task, and has required significant resources from the Pangeo Project up to this point.
+
+## The Pangeo-2i2c Partnership
+
+[2i2c](https://2i2c.org) is a non-profit team that develops and operates cloud infrastructure for interactive computing workflows.
+We have extensive experience in Jupyter workflows in the cloud and a long history of contributions to projects in this ecosystem.
+We have built a cloud deployment management system that allows us to centralise and configure the deployment of many independent JupyterHubs, empowering communities to leverage the same infrastructure (and team!) for JupyterHubs running in the cloud.
+
+Similarly to Pangeo, all of 2i2c's core infrastructure is cloud- and vendor-agnostic, and follows a model of building open source tools and giving back to those communities.
+Our partnership with Pangeo began through 2i2c's core competency in these areas and the similarity between the two project's technical stacks.
+
+We've begun a partnership whereby 2i2c will manage Pangeo's cloud infrastructure and lead efforts to develop new features, in partnership with open source communities.
+We sketched out a few ideas to focus on in this [kick-off thread on Discourse](https://discourse.pangeo.io/t/notes-from-the-pangeo-2i2c-kick-off-meeting/1587).
+This approach allows each community to focus on it's core strengths: Pangeo will continue to grow an open community and scientific software ecosystem around geospatial analytics, and 2i2c will oversee the development and operations of the core cloud infrastructure stack that powers Pangeo's workflows.
+
+We are extremely grateful to the Pangeo project for giving us the opportunity to serve their community, and we look forward to a long partnership ahead! :rocket:
```

Comment:
```suggestion
In some areas we are still experiment with different collaboration models to ensure that the needs of the Pangeo community are met in a way that is also sustainable for 2i2c. Over the coming weeks, you may see some conversations (and threads for feedback!) about different support and operations models that work best for the community. We are excited to use this as an opportunity to learn more about how to serve more complex and diverse communities like Pangeo.

We are extremely grateful to the Pangeo project for giving us the opportunity to serve their community, and we look forward to a long partnership ahead! :rocket:
```

How about this language, and a plan like the following:

- Post this blog post
- As a stop-gap, @sgibson91 keep an eye on the Pangeo Discourse for issues that seem related to 2i2c SRE.
- Over the next few weeks we work out a model for balancing `support@2i2c.org` and @sgibson91's role regarding operations for the Pangeo hubs (or, whoever else is primarily working with Pangeo if we bring another person to supplement this role when she moves to partial EOSS funding)

Does that work?

Reply:
Yep, sounds good to me! 🤞🏻 there aren't many more issues - I think the netpol stuff is at least put to bed.